src/main/java/seedu/pennywise/model/entry/Tag.java

```diff
@@ -18,22 +15,64 @@ public class Tag {
     public static final String EXPENDITURE_CONSTRAINTS = "Expenditure tag must only be one of the following: \n"
             + "Food, Groceries, Entertainment, Transport, Education, Housing, Others";
```

Comment:
I think we can apply a stream on `ExpenditureTag`to avoid writing the raw tag strings in the future.

Reply:
added a additional method to the enum instead of using stream, would that be ok?